tests/addUserFuncPart1.spec.ts

```diff
@@ -0,0 +1,72 @@
+import { test, expect } from "@playwright/test";
+import { generateRandomUserName } from "../helpers/generateRandomUserName";
+import { GenderOptions } from "../enums/GenderOptions";
+const validUserData = [
+  {
+    userNameValue: "nб3-w",
+    yearOfBirthValue: "1900",
+    genderValue: GenderOptions[0],
+  },
+  {
+    userNameValue: "йцу",
+    yearOfBirthValue: "2005",
+    genderValue: GenderOptions[1],
+  },
+  {
+    userNameValue: "new user",
+    yearOfBirthValue: "2004",
+    genderValue: GenderOptions[2],
+  },
+
+  // TODO: uncomment after bugfix:
```

Comment:
Let it be, but it will be better to point here the name of the ticket, I mean part with digits. 
For example: CORE-5421